main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player in data:
+        race_name = data[player]["race"]["name"]
+        race_description = data[player]["race"]["description"]
+        Race.objects.get_or_create(name=race_name,
+                                   description=race_description)
```

Comment:
This is a performance issue. You are making a database query inside a loop. This could lead to a large number of queries if there are many players, which could slow down your program. Consider using `bulk_create` to create multiple objects in one query. However, you will need to handle the case where the race already exists in the database yourself.